crates/aptos-crypto/src/p256_ecdsa/p256_sigs.rs

```diff
@@ -96,6 +97,15 @@ impl P256Signature {
         let new_sig = p256::ecdsa::Signature::from_scalars(r, new_s_nonzero).unwrap();
         P256Signature(new_sig)
     }
+
+    /// If signature bytes are serialized correctly, this function will return a canonical signature
+    /// that passes malleability checks.
+    pub fn make_canonical_from_bytes_unchecked(
```

Comment:
inline?

Reply:
It is modeled after the `make_canonical` function above and not going to be used for verification, only testing

Reply:
if it's for test only can we `cfg` it ?

Reply:
Sure!